crate/kmip/src/crypto/wrap/tests.rs

```diff
@@ -154,7 +173,21 @@ fn test_encrypt_decrypt_rfc_5649() {
 #[test]
 #[cfg(not(feature = "fips"))]
 fn test_encrypt_decrypt_rfc_ecies_x25519() {
-    let wrap_key_pair = create_x25519_key_pair("sk_uid", "pk_uid").unwrap();
+    use crate::kmip::kmip_types::CryptographicUsageMask;
```

Comment:
and remove this one

Reply:
fixed in commit ab48ee823b87dd914cdb2b71a7609bbc8a552bf9